schemas/org.whatwg/media_element/jsonschema/1-0-0

```diff
@@ -0,0 +1,274 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema for a media player event",
+  "self": {
+    "format": "jsonschema",
+    "name": "media_element",
+    "vendor": "org.whatwg",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "autoPlay": {
+      "type": "boolean",
+      "description": "If playback should automatically begin as soon as enough media is available to do so without interruption"
+    },
+    "buffered": {
+      "description": "An array of time ranges that have been buffered",
+      "items": {
+        "type": "object",
+        "description": "A time range object",
+        "properties": {
+          "start": {
+            "description": "The beginning of the time range",
+            "maximum": 9007199254740991,
+            "minimum": 0,
+            "type": "number"
+          },
+          "end": {
+            "description": "The end of the time range",
+            "maximum": 9007199254740991,
+            "minimum": 0,
+            "type": "number"
+          }
+        },
+        "additionalProperties": false
+      },
+      "type": "array"
+    },
+    "controls": {
+      "type": "boolean",
+      "description": "If the user agent should provide it's own set of controls"
+    },
+    "crossOrigin": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "CORS settings value of the media player",
+      "maxLength": 255
+    },
+    "currentSource": {
+      "type": "string",
+      "description": "The absolute URL of the media resource",
+      "maxLength": 65535
+    },
+    "defaultMuted": {
+      "type": "boolean",
+      "description": "If audio is muted by default"
+    },
+    "defaultPlaybackRate": {
+      "type": "number",
+      "description": "The default media playback rate of the player",
+      "minimum": -9007199254740991,
+      "maximum": 9007199254740991
+    },
+    "disableRemotePlayback": {
+      "type": [
+        "boolean",
+        "null"
+      ],
+      "description": "If the media element is allowed to have a remote playback UI"
+    },
+    "error": {
+      "type": [
+        "object",
+        "null"
+      ],
+      "description": "An object of the latest error to occur, or null if no errors"
+    },
+    "fileExtension": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "The media file format",
+      "enum": [
+        "aac",
+        "aacp",
+        "caf",
+        "flac",
+        "mp3",
+        "mp4",
+        "ogg",
+        "wav",
+        "webm"
```

Comment:
I think we might want to loosen this, what happens when a new format comes along? We'll have to update this schema and I don't think this enum check adds too much to this.

Reply:
Enum removed.